Python/optimizer_bytecodes.c

```diff
@@ -980,6 +984,23 @@ dummy_func(void) {
         }
     }
 
+    op(_GUARD_NOS_NULL, (null, unused -- null, unused)) {
+        if (sym_is_null(null)) {
+            REPLACE_OP(this_instr, _NOP, 0, 0);
+        }
+        sym_set_null(null);
+    }
+
+    op(_GUARD_CALLABLE_TYPE_1, (callable, unused, unused2 -- callable, unused, unused2)) {
```

Comment:
I am not entirely sure why I cannot do this here:
```suggestion
    op(_GUARD_CALLABLE_TYPE_1, (callable, unused, unused -- callable, unused, unused)) {
```
I get `SyntaxError: Duplicate name unused`, though `bytecodes.c` allows it :thinking:

Reply:
Weird. Maybe `unused` isn't a meaningful value here like it is in `bytecodes.c` (maybe @markshannon knows)?

Reply:
We can leave it like this. I'll create an issue for Mark to update the cases generator and mention this line.

Reply:
ok!

Reply:
Appears to be fixed now thanks to https://github.com/python/cpython/pull/132615, I updated the PR